python/chronos/example/serving/data

```diff
@@ -0,0 +1 @@
+1422738000.0, 1422739800.0, 1422741600.0, 1422743400.0, 1422745200.0, 1422747000.0
```

Comment:
why the data here so large? I remember the actual value here was somewhere 5000~30000.

Reply:
If we use scaler, should we provide a data that is scaled here? maybe something around 0.

Reply:
yeah, the data before is not correct. Now, the data is generated by:
```python
scaler = StandardScaler()
train_data, val_data, test_data = get_public_dataset("nyc_taxi", with_split=True, test_ratio=0.1)

for data in [train_data, val_data, test_data]:
     data.scale(scaler, fit=data is train_data).roll(lookback=6, horizon=2)

x_test, y_test = test_data.to_numpy()
data = x_test[0]
```